lib/up_to_date_checker.js

```diff
@@ -0,0 +1,79 @@
+'use strict'
+const ignore = require('ignore')
+
+/**
+ * Check branch up to date with master if specific files are modified in the PR
```

Comment:
```suggestion
 * Check branch is up to date with master if specific files are modified in the PR
```

Reply:
I rephrased the docstring. I hope it's more readable right now.

Reply:
I rephrased the docstring. I hope it's more readable right now.